### PR TITLE
Add Outdated k6 Release Used to Performance Insights

### PR DESCRIPTION
**Pull request by ana-yankova**
Documentation for the new performance insight `Outdated k6 Release Used`. The recommendations are exactly the same as the one we display in the app and are curtesy of @nicolevanderhoeven

### CODE REVIEW

**Review by 2Steaks — APPROVED** [review]
LGTM! 🚀

File: src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md [hunk 1]
```diff
@@ -215,6 +216,17 @@ for (let id = 1; id <= 1000; id++) {
...
+### Outdated k6 release used
+
+*Identifier*: `best_practice_outdated_k6_release_used`
+- **Happens when**:
+  you use a legacy version of k6 that is significantly older than the latest stable version.
+- **Recommendations**:
+  - Download and install the latest release of k6 [here](/getting-started/installation) or upgrade your packages if relevant.
+  - Update the k6 binary that your CI/CD pipeline uses for running tests.
+  - If you're part of an organization or team, collectively decide on a version of k6 to use going forward for consistency and ease of comparison.
+  - If you have questions about which version of k6 to use, please contact our [Support team](mailto:support@k6.io).
```
Comment (2Steaks) [inline src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md:228]: I think this line is more for in-app support

**Comment by ana-yankova** [comment]
@MattDodsonEnglish Could you have a quick look? We are releasing the feature tomorrow and would like to have the docs ready.

File: src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md [hunk 2]
```diff
@@ -215,6 +216,16 @@ for (let id = 1; id <= 1000; id++) {
 
 </CodeGroup>
 
+### Outdated k6 release used
+
+*Identifier*: `best_practice_outdated_k6_release_used`
+- **Happens when**:
+  you use a legacy version of k6 that is significantly older than the latest stable version.
+- **Recommendations**:
+  - Download and install the latest release of k6 [here](/getting-started/installation) or upgrade your packages if relevant.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md:225]:
```suggestion
  - [Install the latest release of k6](/getting-started/installation), or upgrade your existing packages.
```

File: src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md [hunk 3]
```diff
@@ -215,6 +216,16 @@ for (let id = 1; id <= 1000; id++) {
...
+  - Update the k6 binary that your CI/CD pipeline uses for running tests.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md:226]:
```suggestion
  - Update the k6 binary that your CI/CD pipeline uses to run tests.
```

**Review by MattDodsonEnglish — APPROVED** [review]
Looks good! I just made the link text more descriptive (part of Google's style guide[^1]) and used an infinitive instead a Gerund(because it makes the text two characters shorter :-) )

[^1]: https://developers.google.com/style/link-text

**Pull request merged**